go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -46,9 +49,11 @@ type topologyWatcherFactoryFunc func(topoServer *topo.Server, hc discovery.Healt
 type throttlerFactoryFunc func(name, unit string, threadCount int, maxRate int64, maxReplicationLagConfig throttler.MaxReplicationLagModuleConfig) (ThrottlerInterface, error)
 
 var (
-	healthCheckFactory     healthCheckFactoryFunc
-	topologyWatcherFactory topologyWatcherFactoryFunc
-	throttlerFactory       throttlerFactoryFunc
+	healthCheckFactory       healthCheckFactoryFunc
+	topologyWatcherFactory   topologyWatcherFactoryFunc
+	throttlerFactory         throttlerFactoryFunc
+	topoCellsRefreshInterval = time.Minute
```

Comment:
I think either we make it configurable or we keep this at a higher value. A minute seems very less to do a topo call for cell change.

Reply:
@harshit-gangal sounds good, adding a command line flag for this 👍